lib/buckets/common.js

```diff
@@ -74,7 +74,7 @@ function _list(type, req, bucket_id) {
         format('invalid _list type: %s', type));
     assert.object(req, 'req');
     assert.object(req.log, 'req.log');
-    assert.object(req.params, 'req.params');
+    assert.object(req.query, 'req.query');
     // assert.optionalUuid does not behave properly
```

Comment:
I wonder if this behaves better with the newer version.

Reply:
Is there particular behaviour you mean?

I generally would advocate never using `req.params` in restify code: One should use `req.query` and/or `req.body` to be explicit.

Reply:
I was referring to misbehavior of `assert.optionalUuid`. It the old version it was throwing asserts when a valid UUID was provided.

Reply:
I haven't bumped `assert-plus` as part of this PR.  I'm only trying to bump versions that are strictly required for node v6.  I want to do a follow-up PR after this one is merged where certain deps are updated (like assert-plus).

Reply:
Got it. Sorry, should have double-checked that before commenting.